### PR TITLE
Renamed Warden's 'Protect' ability, updated its and other ability icon states, added disclaimers and flavor text to all strains

### PR DESCRIPTION
**Pull request by Ultimate-Fluff**
# About the pull request
Warden's 'Protect' ability made no sense because it no longer can shield another xeno, so I renamed the ability to 'Assist', changed the flavor text, and I also created a new icon state for it.
image

In addition, Lurker's ability icon for invisibility incorrectly portrays a human instead of a xenomorph turning invisible, so I fixed that, too. 
image

Some ability icon states were tweaked to conform to the standards we expect of our sprites, including Pounce, Boiler's Toggle-Long-Range, and others. Here are a few examples:
image
image
image
image
image
image

Also modified added icon states for reinforced structures, including a shiny new ability icon state for the reinforced door!
image
image
image

For the code portion, I added flavor text to all strains. Check it out! 

The complete list of changes is in the changelog, but these are the most major ones. 

# Explain why it's good for the game
Everything is more uniform and some ability icons represent their abilities much better than before.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: Puckaboo2
code: Renamed Warden's 'Protect' ability to 'Assist' and fixed its flavor text to better represent its function.
code: Any xenomorph strain description that previously did not completely disclose any abilities or attributes that will be removed or changed upon mutating will now disclose them, often within the first sentence.
add: Every xenomorph strain now has flavor text, personalized as a message from the Queen, herself.
imageadd: Added a new ability icon for Warden's aforementioned 'Assist' ability.
tweak: Updated a handful of ability icons to conform to the higher-quality standard we expect from our sprites.
tweak: Any ability icon that did not have a border or did not share the same border format as the current standard now has its correct border palette.
tweak: The ability icons for reinforced resin walls, doors, and membranes now have separate sprites from their normal counterparts.
tweak: Centered some ability icons.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm
Comment (Zonespace27) [inline code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm:304]: Remove them if they are

Comment (Ultimate-Fluff) [reply code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm:304]: I'm a bit afraid to; I don't want to completely remove portions of code unless I get the approval of a maintainer.

Edit: I tried, but then something broke so I am leaving it alone.

Comment (Zonespace27) [reply code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm:304]: If it causes errors then it's needed

Comment (fira) [reply code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm:304]: As Zonespace says. Either remove them or remove that line, there's no real point to just add a "maybe" comment

**Review by fira — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm
Comment (Ultimate-Fluff) [reply code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm]: Fixed.

**Comment by zeroisthebiggay** [comment]
https://camo.githubusercontent.com/816c6bb5eaba4953e4a2afca0625af29d8ea389497ecf3cdc9db91e35e28e1b0/68747470733a2f2f696462322e616666656374656461726330372e636f2e756b2f696d616765732f32363339313036312f323439362f323235373038323634363930343038353537372e676966
xenohide is kinda schmooving

**Comment by Ultimate-Fluff** [comment]
> 68747470733a2f2f696462322e616666656374656461726330372e636f2e756b2f696d616765732f32363339313036312f323439362f323235373038323634363930343038353537372e676966 [ 68747470733a2f2f696462322e616666656374656461726330372e636f2e756b2f696d616765732f32363339313036312f323439362f323235373038323634363930343038353537372e676966 ](https://user-images.githubusercontent.com/43571008/215851013-928991d7-7cc3-4d24-9435-719764616976.gif) [ ](https://user-images.githubusercontent.com/43571008/215851013-928991d7-7cc3-4d24-9435-719764616976.gif) xenohide is kinda schmooving

Fixed it, thanks!

**Review by stanalbatross — APPROVED** [review]
code good

**Review by kugamo — CHANGES_REQUESTED** [review]
See https://discord.com/channels/150315577943130112/746325423289401395/1071230453777965116

**Comment by kugamo** [comment]
The sheer number of sprites changed on this PR makes reviewing extremely tedious, all for very minor changes. Closing until you have the GBP to warrant it. (feel free to atomize out just the description + non border sprite changes in a separate PR)

**Pull request closed without merging**
